tests/chart/test_auth_sidecar.py

```diff
@@ -89,19 +90,26 @@ def test_auth_sidecar_config_with_dag_server_enabled(self, kube_version):
             show_only=[
                 "templates/dag-deploy/dag-server-statefulset.yaml",
                 "templates/dag-deploy/dag-server-service.yaml",
+                "templates/dag-deploy/dag-server-auth-sidecar-configmap.yaml",
             ],
         )
 
-        common_dagserver_sts_test_cases(docs, 2)
+        common_dagserver_sts_test_cases(docs, 3)
         c_by_name = get_containers_by_name(docs[0])
         assert c_by_name["auth-proxy"]["resources"] == resources
         assert volumeMounts in c_by_name["auth-proxy"]["volumeMounts"]
 
         assert docs[1]["kind"] == "Service"
         assert docs[1]["apiVersion"] == "v1"
         assert docs[1]["metadata"]["name"] == "release-name-dag-server"
+        print(docs[2]["data"]["nginx.conf"])
         assert authSidecarServicePorts in docs[1]["spec"]["ports"]
 
+        nginx_conf = pathlib.Path(
+            "tests/chart/test_data/dag-server-authsidecar-nginx.conf"
```

Comment:
If we're going to start doing this kind of thing, we'll need to come up with a way to organize these files better by naming them after the function that uses them, or by their content hash. We don't need to solve this now if we don't want to, but if we don't solve it now we'll have to solve it as soon as we start adding more files.